library/build.gradle

```diff
@@ -20,6 +25,8 @@ static def getLibraryInfo() {
 }
 
 dependencies {
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
```

Comment:
Any reason why you are sticking with jdk 7?

Reply:
We won't get any additional functionality by updating to 8. (I also can't remember if that would force implementing apps to use JDK8 as well)